lib/ex_doc.ex

```diff
@@ -1,3 +1,20 @@
+defmodule ExDoc.Default do
```

Comment:
the reason is that `ExDoc` module depends on  `ExDoc.Config`, and `ExDoc.Config` depends on the default config function. I tried creating `ExDoc.Config.deafult/0-1` functions an undefined function error. that was why,
If you can find a way that this will work without the extra module, please let me know.. cuz I couldn't